From b6a6c9b842b309f654171f417242f8cfffe6460d Mon Sep 17 00:00:00 2001
From: Alejandro Abdelnur <tucu@cloudera.com>
Date: Fri, 25 Jan 2013 09:56:46 -0800
Subject: [PATCH 0901/1357] HADOOP-8857. hadoop.http.authentication.signature.secret.file docs should not state that secret is randomly generated. (tucu)

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/trunk@1438601 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 55f0d031bea2da7a0e0772fc07d556c87ffe8201)

Conflicts:
	hadoop-common-project/hadoop-common/CHANGES.txt
	hadoop-common-project/hadoop-common/src/site/apt/HttpAuthentication.apt.vm
---
 .../content/xdocs/HttpAuthentication.xml           |    4 ++--
 .../src/main/resources/core-default.xml            |    1 -
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/docs/src/documentation/content/xdocs/HttpAuthentication.xml b/hadoop-common-project/hadoop-common/src/main/docs/src/documentation/content/xdocs/HttpAuthentication.xml
index fc7b59b..d6e9f62 100644
--- a/hadoop-common-project/hadoop-common/src/main/docs/src/documentation/content/xdocs/HttpAuthentication.xml
+++ b/hadoop-common-project/hadoop-common/src/main/docs/src/documentation/content/xdocs/HttpAuthentication.xml
@@ -83,8 +83,8 @@
       </p>
 
       <p><code>hadoop.http.authentication.signature.secret.file</code>: The signature secret 
-      file for signing the authentication tokens. If not set a random secret is generated at 
-      startup time. The same secret should be used for all nodes in the cluster, JobTracker, 
+      file for signing the authentication tokens. 
+      The same secret should be used for all nodes in the cluster, JobTracker, 
       NameNode, DataNode and TastTracker. The default value is 
       <code>${user.home}/hadoop-http-auth-signature-secret</code>.
       IMPORTANT: This file should be readable only by the Unix user running the daemons.
diff --git a/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml b/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
index f00b5f3..1c4a82f 100644
--- a/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
+++ b/hadoop-common-project/hadoop-common/src/main/resources/core-default.xml
@@ -904,7 +904,6 @@
   <value>${user.home}/hadoop-http-auth-signature-secret</value>
   <description>
     The signature secret for signing the authentication tokens.
-    If not set a random secret is generated at startup time.
     The same secret should be used for JT/NN/DN/TT configurations.
   </description>
 </property>
-- 
1.7.0.4

